libultraship/libultraship/Window.cpp

```diff
@@ -655,6 +655,11 @@ namespace Ship {
 	    return APlayer;
     }
 
+    void Window::ReinitAudioPlayer() {
+        audioBackend = GetConfig()->getString("Window.AudioBackend");
+        InitializeAudioPlayer();
+    }
```

Comment:
is there a way to do this outside of libultraship? i'm not sure if it's the best spot for it, and it'll take a little while before we can start getting LUS changes in again

Reply:
I can remove that one but the variables **audioBackend** and **APlayer** are private and cannot be get/set outside LUS, so I'll reedit it again by adding a string parameter in **InitializeAudioPlayer**, and/or remove audioBackend var entirely. 

*This includes both the function and variable for the window manager **InitializeWindowManager** and gfxBackend for the sake of clarity.*